### ISSUE TITLE
The extension causes Visual Studio to freeze

### ISSUE THREAD

**Issue by thomaslevesque** [mariusbancila/memefulcomments#1]
The extension causes Visual Studio to freeze as soon as I open an editor. Tried it 3 times, got the same result 3 times. Everything went back to normal when I disabled it.

**Comment by mariusbancila** [comment]
I was able to reproduce from time to time and I believe also fix it. I can no longer get Visual Studio to freeze, with the new version 1.2.

Could you try this new version and confirm if it works well, or still have the problem?

**Comment by thomaslevesque** [comment]
Thanks! It seems to work, but I tried on a different machine... I'll try again at work tomorrow, since the problem was easily reproducible.

BTW, I noticed that you're using the WpfAnimatedGif library, which I maintain. Be aware that it uses an awful lot of RAM... if it becomes an issue, I have a different library ([XamlAnimatedGif](https://github.com/XamlAnimatedGif/XamlAnimatedGif/)), which uses less RAM (but more CPU).
